pom.xml

```diff
@@ -479,8 +473,8 @@ from system library in Java 11+ -->
             </dependency>
             <dependency>
                 <groupId>org.glassfish</groupId>
-                <artifactId>javax.json</artifactId>
-                <version>1.1.4</version>
+                <artifactId>jakarta.json</artifactId>
+                <version>1.1.6</version>
             </dependency>
```

Comment:
I use this new code:
```
            <dependency>
                <groupId>jakarta.json</groupId>
                <artifactId>jakarta.json-api</artifactId>
                <version>2.1.3</version>
            </dependency>
            <dependency>
                <groupId>org.eclipse.parsson</groupId>
                <artifactId>jakarta.json</artifactId>
                <version>1.1.6</version>
            </dependency>
```

Reply:
Sorry, you missed the point: this is not a pull request to push the dependencies to latest version. I just updated the artifact and group id to the new used names without breaking any compatibility with the current code and provide a way to use renamed JavaEE namespace in future.  This is only one step in a list of many steps and not the end.